validation/superchain-version_test.go

```diff
@@ -55,42 +55,31 @@ func TestSuperchainWideContractVersions(t *testing.T) {
 	}
 }
 
-func TestContractVersions(t *testing.T) {
+func testOPChainMatchesATag(t *testing.T, chain *ChainConfig) {
```

Comment:
Suggestion: change name to `testContractsMatchATag` for more clarity

Reply:
[f76a4ee](https://github.com/ethereum-optimism/superchain-registry/pull/350/commits/f76a4eee39ea333ec9fa1c7f09ba54c2824157fe)